static/skin/index.js

```diff
@@ -1,31 +1,65 @@
 const root = $(`link[type='root']`).attr('href');
+let viewPortHeight = window.innerHeight;
+let isFetching = false;
+let isEnd = false;
+const basicConfig = {
```

Comment:
The name of this variable is not a good one

Reply:
@veloman-yunkan what do you suggest would be a better name?
also, what should be done about the risk of interference with other JS files should I add a prefix like KSUI(for kiwix serve UI)?
naming would be then like KSUI_isFetching etc.

Reply:
> what do you suggest would be a better name?

`incrementalLoadingParams`?

Reply:
Regarding the risk of interference, @kelson42 already pointed out that we don't have such an issue currently. The general approach is to hide the variables in closures.

Reply:
ok then I am not adding any closure for now.